test/clojisr/v1/help_test.clj

```diff
@@ -0,0 +1,30 @@
+(ns clojisr.v1.help-test
+  (:require 
+   [clojure.string :as str] 
+   [clojure.test :refer [is deftest]]
+   [clojisr.v1.r  :as r]))
+
+(r/require-r '[randomForest])
```

Comment:
Let's use only default packages for tests.

Reply:
see last commit